README.md

```diff
@@ -1,9 +1,9 @@
 # FINALE
 
-FINALE is a lightweight web app to view GCode. FINALE stands for FINALE Is Not A LabVIEW Editor.
+FINALE is a lightweight WebApp to view G Code. FINALE stands for FINALE Is Not A LabVIEW Editor.
 
 This solves many use cases like:
-  - Code sharing : Sharing code with a person with no LV license necessitates printing/taking screenshot.
+  - Code sharing: Sharing code with a person with no LV license necessitates printing/taking screenshot.
   - Viewing VIs saved in incompatible version.
```

Comment:
Might want to just say "LabVIEW code" instead of saying "VIs". You could add another sentence saying this includes VIs, classes, typedefs, etc.

Reply:
Fixed.